packages/frontend/src/hooks/useColumnTotals.ts

```diff
@@ -13,12 +16,28 @@ type Args = {
     itemsMap: Record<FieldId, Field | TableCalculation>;
 };
 
+const isSummable = (item: Field | TableCalculation) => {
+    if (isField(item) || isAdditionalMetric(item)) {
+        const numericTypes: string[] = [
+            DimensionType.NUMBER,
+            MetricType.NUMBER,
+            MetricType.COUNT,
+            MetricType.COUNT_DISTINCT,
+            MetricType.SUM,
+            MetricType.MIN,
+            MetricType.MAX,
```

Comment:
We should remove min and max as well 🤔 what do you think ?

Reply:
Mmmm, you're probably right. @TuringLovesDeathMetal can you think of a case where MIN/MAX can be useful on totals ?

Reply:
not really! Unless you're using min/max as a hack for getting a distinct value from a grouped dimension. 

let's remove it and then if someone complains/finds a use case for it, we can add it back :)